pdns/dnsdistdist/docs/changelog.rst

```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+.. changelog::
+  :version: 1.9.1
+  :released: 14th of March 2024
+
+  This release does not contain any dnsdist code changes compared to 1.9.0.
+  The only thing that changed is the version of Quiche, because of a `security update <https://github.com/cloudflare/quiche/releases/tag/0.20.1>`_.
+
+  Please review the :doc:`Upgrade Guide <../upgrade_guide>` before upgrading.
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 13909
```

Comment:
I never remember if we want to refer to the backport PR or the master one. This is the master.

Reply:
For me it sometimes even depends on whether I have the backport number ready when writing the changelog, and I also don't think it matters much, but I'll change it to the backport one.

Reply:
I usually put the backport PR but it doesn't matter much AFAICT